doc/git-etiquette.md

```diff
@@ -0,0 +1,65 @@
+# Git Etiquette Guide
+
+To ensure smooth collaboration, we've outlined the following Git etiquette guide. This guide will help in writing concise commit messages, managing git branches, pull request descriptions, and other best practices.
+
+## 1. Commit Messages
+
+Your commit message should provide clear context about the change. This allows others (and future you) to understand why a particular change was made.
+
+Here are the guidelines to write concise and meaningful commit messages:
+
+- **Use the Imperative Mood**: Commit messages should be written in the present tense and imperative mood ("Add feature" not "Added feature" or "Adds feature").
+- **First Line Should Be Summary**: The first line of your commit message should be a brief summary of the changes, kept under 50 characters whenever possible.
+- **Body**: Follow this summary line with a blank line, then a more detailed explanation is always helpful but not necessary. This is often useful when the commit has significant importance or complexity.
+- **Consider Including the 'Why'**: If the commit resolves an issue or bug, mention the issue number and provide a brief explanation of how the changes fix the issue.
+
+Here's a good commit message example:
+
+```
+Fix off-by-one error in loop (issue #123)
```

Comment:
I personally don't find this example to be a good commit message 🙃 It's extremely vague on where/how this affects the codebase at all.
Perhaps we can just look through our commit history and take some good examples from there??

Here are some I found that I think are clear/insightful:

_Add flash toggle for the camera in sync screen_ 
- we know what exactly was added and what screens/part of the codebase it relates 

_Fix muting channel from inside and outside chat page_ 

_feat: implement onboarding modal transition for sign in screen_

Reply:
ah yes this was just a dummy one.
I do plan to pick examples from our codebase

Reply:
cool :)